7.1/Dockerfile

```diff
@@ -21,6 +20,7 @@ RUN apt-get -qq update > /dev/null && DEBIAN_FRONTEND=noninteractive apt-get -qq
     php7.1-intl \
     php7.1-fpm \
     php7.1-xml \
+    php7.1-mbstring \
```

Comment:
If we add mbstring by default, we need to add it to 5.6 image too, and update README.md https://github.com/kibatic/symfony-docker#minimal-package-included